city_scrapers/spiders/cook_justice_advisory.py

```diff
@@ -0,0 +1,196 @@
+import re
+from collections import defaultdict
+from datetime import datetime
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+
+
+class CookJusticeAdvisorySpider(CityScrapersSpider):
+    name = "cook_justice_advisory"
+    agency = "Cook County Justice Advisory"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.cookcountyil.gov"]
+
+    def __init__(self, *args, **kwargs):
+        self.agenda_map = defaultdict(list)
```

Comment:
Looking at this again, it looks like there are minutes as well as agendas, and we'll want to reflect that if we can here.